docs/source/extend_kedro/plugins.md

```diff
@@ -29,13 +29,12 @@ def to_json(metadata):
     pipeline = pipelines["__default__"]
     print(pipeline.to_json())
 ```
+From version 0.18.14, Kedro replaced `setup.py` with `pyproject.toml`. The plugin needs to provide entry points in either file. If you are using `setup.py`, please refer to the [`0.18.13` version of documentations](https://docs.kedro.org/en/0.18.13/extend_kedro/plugins.html).
 
-The plugin provides the following `entry_points` config in `setup.py`:
-
-```python
-setup(
-    entry_points={"kedro.project_commands": ["kedrojson = kedrojson.plugin:commands"]}
-)
+To add the entry point to `pyproject.toml`, the plugin needs to provide the following `entry_points` configuration:
+```toml
+[project.entry-points."kedro.project_commands"]
+kedrojson = kedrojson.plugin.commands
```

Comment:
oops, typo

Reply:
I did test it with the plugin hooks🥲 maybe I mess it up when I  copy-paste it back to the doc. Thanks for spotting this.